manifests/0000_31_cluster-baremetal-operator_06_deployment.yaml

```diff
@@ -76,6 +80,7 @@ spec:
           name: config
         - mountPath: /etc/tls/private
           name: cluster-baremetal-operator-tls
+      hostNetwork: true
```

Comment:
If we put the CBO on the host network then we need to move a bunch of ports and register them in the port registry.